frontend/src/components/Product.js

```diff
@@ -12,6 +12,19 @@ class Product extends Component {
         <h3 className="product-name">{this.props.name}</h3>
```

Comment:
we should let the higher level component pass in 'product' as props, intead of passing in (product.id, product.price, product.name, product.src) individually.